conf/ocsci/acm_hub_downstream_image.yaml

```diff
@@ -0,0 +1,3 @@
+---
+ENV_DATA:
+  acm_downstream_image: "2.4.3-DOWNSTREAM-2022-04-13-07-05-00"
```

Comment:
Any reason for hardcoded values? Also, it's for 2.4.3 we need for 2.5

Reply:
@prsurve how we will match acm version? With OCP? Like in ocp 4.10 we will use 2.5 and in 4.11 different version? I thought about to add it in our ocp version config files.

Reply:
This is really just an example config and not one that will be used by jenkins. I chose this version because this is the last RC version, where as 2.5 does not have one yet.

@petr-balogh I have not talked to them about adding a floating tag. I can bring this up with them but we probably can't count on having this in the short term. I expected that we would add a field to the jenkins deployment job to specify the tag that we wish to deploy. 

Also, I spoke to them about compatibility and was given some information on which versions are supported for which OCP versions. This version works for any version we are currently actively testing with acm hub (4.9 and up).

Reply:
For now, we can use the 2.5 build

Reply:
I have move this file to the `examples` dir and updated to the latest 2.5 image. @petr-balogh I also brought up the topic of `latest-x.y` tags and have started that discussion. Will update you on it.